### PR TITLE
auv-msgs update 

### PR DESCRIPTION
**Pull request by IgnacioCarlucho**
@bmagyar @decabyte @tfurf @henriqueribeiro @narcispr @dulanad @brunovmenna

Guys, this tries to address some of the comments made on #3 
I have tried to keep the changes minimal, changing what most of you agreed on. 
Let me know what you think and we take it from there.

### CODE REVIEW

**Comment by bmagyar** [comment]
Could you please sum up your changes and new additions as a recap?

**Comment by IgnacioCarlucho** [comment]
Bence, yes sorry,  you are right.
The mayor changes in this version are: 

*  All the Req msgs were renamed to Request, as suggested by @tfurf, to give a better idea of what the messages are, for instance now we have _BodyVelocityRequest_  
* Added a WorldWaypointRequest message. I used the old version that Bence showed in #3, and cleaned it a bit, as was suggested in the discussion. I tried to keep it simple and made it similar to the other Request messages. Basically updated the comments and removed the goal descriptor. And also remade _max_speed_percent_ to be a 6dof velocity limit, and rename it _twist_limit_ (do you think this is descriptive?) 
* I also updated the maintainers of the package. 

That is all. what do you guys think?

**Review by bmagyar — APPROVED** [review]

**Comment by bmagyar** [comment]
As this breaks API for everyone using it we'll have to increase major version number on the release & send an announcement on ROS discourse. I've just approved the changes, unless there are others with objections you are good to merge this.

**Pull request merged**
